libvast/src/system/exporter.cpp

```diff
@@ -384,15 +384,15 @@ behavior exporter(stateful_actor<exporter_state>* self, expression expr,
           self->send(x, atom::exporter_v, self);
     },
     [=](atom::run) {
-      VAST_INFO(self, "executes query:", to_string(self->state.expr));
+      VAST_VERBOSE(self, "executes query:", to_string(self->state.expr));
       self->state.start = system_clock::now();
       if (!has_historical_option(self->state.options))
         return;
       self->request(self->state.index, infinite, self->state.expr)
         .then(
           [=](const uuid& lookup, uint32_t partitions, uint32_t scheduled) {
-            VAST_DEBUG(self, "got lookup handle", lookup, ", scheduled",
-                       scheduled, '/', partitions, "partitions");
+            VAST_VERBOSE(self, "got lookup handle", lookup, ", scheduled",
+                         scheduled, '/', partitions, "partitions");
```

Comment:
Are related messages that print status updates also VERBOSE?

Reply:
Some of them are even `INFO` (see below), some are `DEBUG`. I think VERBOSE didn't really exist when most of this was written, so there's not a lot of consistency about this.